### DISCUSSION TITLE
[gatsby-plugin-image] Increase the blurriness of blurred placeholder

### DISCUSSION THREAD

**Discussion by mwskwong** [gatsbyjs/gatsby#33524] [Help] [answered]
I find the blurred placeholder of `gatsby-plugin-image` is quite pixelated and I want to make it even blurrier. I notice there is a `blurredOptions` option which is properly what I need, but the document has zero information on what options does it takes.

So how should I make this work?

**Comment by LekoArts** [comment, marked as answer]
Hi!

You can use the `width` to maybe change this:

https://github.com/gatsbyjs/gatsby/blob/9feb3b09dd47892a8d64fa6fa1f0a89ce9de27c2/packages/gatsby-transformer-sharp/src/types.ts#L106-L120

**Reply by mwskwong** [reply]
> Hi!
> 
> You can use the `width` to maybe change this:
> 
> https://github.com/gatsbyjs/gatsby/blob/9feb3b09dd47892a8d64fa6fa1f0a89ce9de27c2/packages/gatsby-transformer-sharp/src/types.ts#L106-L120

I wonder can I increase the "blurriness" instead? After all, it is using `filter: blur(certain_value)` in the CSS.

**Reply by adamatronix** [reply]
Did you end up finding a solution?
